city_scrapers/spiders/chi_northwest_home_equity.py

```diff
@@ -0,0 +1,163 @@
+import re
+from datetime import datetime
+
+import requests
+import scrapy
+from city_scrapers_core.constants import NOT_CLASSIFIED
+from city_scrapers_core.items import Meeting
+from city_scrapers_core.spiders import CityScrapersSpider
+
+
+class ChiNorthwestHomeEquitySpider(CityScrapersSpider):
+    name = "chi_northwest_home_equity"
+    agency = "Chicago Northwest Home Equity Assurance Program"
+    timezone = "America/Chicago"
+    start_urls = ["https://nwheap.com/category/meet-minutes-and-agendas/"]
+
+    def parse(self, response):
+        self.location_dict = {
+            "3022 N. Harlem": "3022 N. Harlem Ave #2S, Chicago, IL 60634",
+            "3022 N. Harlem Ave. #2S": "3022 N. Harlem Ave #2S, Chicago, IL 60634",
+            "3022 Harlem Ave. #2S": "3022 Harlem Ave #2S, Chicago, IL 60634",
+            "3234 N. Central Ave.": "3234 N. Central Ave, Chicago, IL 60634",
+            "5363 W. Lawrence Ave.": "5363 W. Lawrence Ave, Chicago, IL 60630",
+        }
+        self.location_name_dict = {
+            "3022 N. Harlem": "Northwest Home Equity Main Office",
+            "3022 N. Harlem Ave #2S": "Northwest Home Equity Main Office",
+            "3022 Harlem Ave #2S": "Northwest Home Equity Main Office",
+            "3234 N. Central Ave.": "Northwest Home Equity Assurance Program",
+            "5363 W. Lawrence Ave": "Jefferson Park Library",
+        }
+        # Before we begin, need to collect meeting minutes data -- which is
+        #  contained on multiple iterative pages (/page/2 /page/3 etc)
+        r = requests.get("https://nwheap.com/category/meet-minutes-and-agendas/")
```

Comment:
We don't need to use `requests` here since the response from this page should be the first thing Scrapy loads in `start_urls`